From 54e5bf9f3a2a123ae4e0962cfa316ac48230f3fc Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 27 Jul 2016 11:18:28 +0800
Subject: [PATCH] ARM64: dts: rk3399-box: rename files for box-rev1

Make dts files in order as follows:
rk3399.dtsi
rk3399-box.dtsi
rk3399-box-rev1.dts
rk3399-box-rev2.dts
...

Change-Id: I93d1fb4380d0c84a14fc3476b7435b1abe4989d2
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/Makefile         |  2 +-
 .../boot/dts/rockchip/rk3399-box-rev1.dts     | 49 +++++++++++++++++++
 ...9-box-rev2-android.dts => rk3399-box.dtsi} | 12 ++---
 3 files changed, 54 insertions(+), 9 deletions(-)
 create mode 100644 arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-rev2-android.dts => rk3399-box.dtsi} (98%)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 2d2e27f524eb..90947e54783e 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -3,7 +3,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3366-tb.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-r88.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-tb-sheep.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-808-android.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-android.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-cros.dtb
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
new file mode 100644
index 000000000000..4c3c793aef74
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016 Fuzhou Rockchip Electronics Co., Ltd
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively,
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use,
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/dts-v1/;
+#include "rk3399-box.dtsi"
+
+/ {
+	model = "Rockchip RK3399 Board rev1 (BOX)";
+	compatible = "rockchip-box-rev1","rockchip,rk3399-box";
+};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index 734da5ea20c4..d091fd0a50cf 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -39,17 +39,13 @@
  *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
  *     OTHER DEALINGS IN THE SOFTWARE.
  */
-
-/dts-v1/;
-
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/input/input.h>
 #include "rk3399.dtsi"
 #include "rk3399-android.dtsi"
 
 / {
-	model = "Rockchip RK3399 Box Board rev2 (Android)";
-	compatible = "rockchip,rk3399";
+	compatible = "rockchip,rk3399-box","rockchip,rk3399";
 
 	vcc1v8_s0: vcc1v8-s0 {
 		compatible = "regulator-fixed";
@@ -211,8 +207,8 @@
 };
 
 &sdmmc {
-	clock-frequency = <150000000>;
-	clock-freq-min-max = <400000 150000000>;
+	clock-frequency = <100000000>;
+	clock-freq-min-max = <400000 100000000>;
 	supports-sd;
 	bus-width = <4>;
 	cap-mmc-highspeed;
@@ -227,7 +223,7 @@
 };
 
 &sdio0 {
-	clock-frequency = <140000000>;
+	clock-frequency = <100000000>;
 	clock-freq-min-max = <200000 100000000>;
 	supports-sdio;
 	bus-width = <4>;
-- 
2.35.3

